libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -203,6 +205,44 @@ class NavigationRoute internal constructor(
             }
         }
 
+        internal fun create(
```

Comment:
Is this used only for tests? Now it looks like it. Why can't you do all these json manipulations directly in the test-only method?  For example, in `NavigationRouteEx`.
I don't like adding this method to production code because it duplicates directions response structure. And now the knowledge of this structure only lives in mapbox-java, which I think is good and it should stay this way.